src/test/java/brave/opentracing/BraveSpanTest.java

```diff
@@ -136,11 +136,8 @@
     assertThat(spans).hasSize(3);
     assertThat(spans.get(0).traceId).isEqualTo(spans.get(1).traceId)
         .isEqualTo(spans.get(2).traceId);
-    assertThat(spans.get(1).id).isEqualTo(spans.get(2).id);
+    assertThat(spans.get(1).id).isNotEqualTo(spans.get(2).id);
     assertThat(spans.get(0).id).isNotEqualTo(spans.get(1).id);
-
-    assertThat(zipkin.spanStore().getDependencies(System.currentTimeMillis(), null))
```

Comment:
this is why, btw :) though latest zipkin (1.29.0) can link parent+child even across span ids. Maybe bump the zipkin dep and see if this passes?